crates/swc_core/build.rs

```diff
@@ -12,6 +12,15 @@ compile_error!(
      run plugin, use 'plugin_transform_host_*' instead."
 );
 
+#[cfg(all(feature = "__plugin_transform", feature = "common_concurrent"))]
```

Comment:
nit note: I wanted to have more strict checkers, but it'll make cargo's `check / ...` fail when it tries to merge mutually exclusive features. :/